CHANGELOG.md

```diff
@@ -2,10 +2,15 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 # [v1.0.2] (UnReleased)
+### Notes
+ This version of the module supports OneView appliances with version 4.10, using the OneView REST API version 800.
```

Comment:
if you are supporting API 1000, then version should be 4.2 and API version as 1000

Reply:
Changed accordingly.